governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -0,0 +1,411 @@
+import { BN, Wallet } from '@coral-xyz/anchor'
+import { PythCluster } from '@pythnetwork/client/lib/cluster'
+import { useAnchorWallet } from '@solana/wallet-adapter-react'
+import { AccountMeta } from '@solana/web3.js'
+import { getIxPDA } from '@sqds/mesh'
+import { MultisigAccount, TransactionAccount } from '@sqds/mesh/lib/types'
+import copy from 'copy-to-clipboard'
+import { useRouter } from 'next/router'
+import {
+  Dispatch,
+  SetStateAction,
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+import {
+  getProposals,
+  MultisigParser,
+  PythMultisigInstruction,
+  WormholeMultisigInstruction,
+} from 'xc_admin_common'
+import { ClusterContext } from '../../contexts/ClusterContext'
+import { SECURITY_MULTISIG, useMultisig } from '../../hooks/useMultisig'
+import CopyIcon from '../../images/icons/copy.inline.svg'
+import ClusterSwitch from '../ClusterSwitch'
+import Loadbar from '../loaders/Loadbar'
+
+const ProposalRow = ({
+  proposal,
+  setCurrentProposalPubkey,
+}: {
+  proposal: TransactionAccount
+  setCurrentProposalPubkey: Dispatch<SetStateAction<string | undefined>>
+}) => {
+  const status = Object.keys(proposal.status)[0]
+
+  const router = useRouter()
+
+  const handleClickIndividualProposal = useCallback(
+    (proposalPubkey: string) => {
+      router.query.proposal = proposalPubkey
+      setCurrentProposalPubkey(proposalPubkey)
+      router.push(
+        {
+          pathname: router.pathname,
+          query: router.query,
+        },
+        undefined,
+        { scroll: false }
+      )
+    },
+    [setCurrentProposalPubkey, router]
+  )
+  return (
+    <div
+      className="my-2 max-h-[58px] cursor-pointer bg-[#1E1B2F] hover:bg-darkGray2"
+      onClick={() =>
+        handleClickIndividualProposal(proposal.publicKey.toBase58())
+      }
+    >
+      <div className="flex justify-between p-4">
+        <div>{proposal.publicKey.toBase58()}</div>
+        <div
+          className={
+            status === 'active'
+              ? 'text-[#E5D8FE]'
+              : status === 'executed'
+              ? 'text-[#15AE6E]'
+              : status === 'cancelled'
+              ? 'text-[#FF5E00]'
+              : status === 'rejected'
+              ? 'text-[#F54562]'
+              : ''
+          }
+        >
+          {status}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const SignerTag = () => {
+  return (
+    <div className="flex items-center justify-center rounded-full bg-darkGray4 py-1 px-2 text-xs">
+      Signer
+    </div>
+  )
+}
+
+const WritableTag = () => {
+  return (
+    <div className="flex items-center justify-center rounded-full bg-offPurple py-1 px-2 text-xs">
+      Writable
+    </div>
+  )
+}
+
+const Proposal = ({
+  proposal,
+  multisig,
+}: {
+  proposal: TransactionAccount
+  multisig: any
+}) => {
+  const [proposalInstructions, setProposalInstructions] = useState<
+    (PythMultisigInstruction | WormholeMultisigInstruction)[]
```

Comment:
WormholeInstruction are not showing I think :
https://xc-admin-frontend-r30ihde1s-pyth-web.vercel.app/?tab=proposals&cluster=devnet&proposal=9fsFiFafJLSuLCXtqYxLnp5YTYpy4XLtMkuWATAcnmcT